Assignment4/Assignment4.java

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Make sure the class is in a package. Package naming convention should also be followed.